plugins/nv-filters/nvidia-videofx-filter.c

```diff
@@ -348,7 +360,9 @@ static void *nvvfx_filter_create(obs_data_t *settings, obs_source_t *context, en
 	}
 
 	nvvfx_filter_update(filter, settings);
-
+	/* Setup NVIDIA logger */
+	if (nvvfx_new_sdk)
+		vfxErr = NvVFX_ConfigureLogger(NVCV_LOG_ERROR, NULL, &nvvfx_logger_callback, filter);
```

Comment:
Leave a blank line after unenclosed `if` statements, or enclose the `if` with curly braces.